### DISCUSSION TITLE
Suggestion: Show all fields in issues

### DISCUSSION THREAD

**Discussion by nyqvistatwork** [community/community#38735] [Projects and Issues]
# Suggestion

### Issues

- All project fields should be shown by default
- Popup menu should be used for changing what fields to show on issues

### Current view

- Only the field **Status** is being shown by default.
- Custom fields are hidden.
- User action is needed to see custom fields

![image](https://user-images.githubusercontent.com/92366570/200965377-3f1bc99b-2257-47a7-882c-5017bb3da21b.png)

> What if the hidden field is very important for the project?
> What if users could select what fields to show themselves?

### Suggested view

- All fields are shown by default
- No fold/unfold icon
- No hidden +more
- No user action needed to see custom fields
- Maybe a text link to select what fields to show (like labels)?

Something like this but I believe this can also be reworked to give the fields a clearer look.

![image](https://user-images.githubusercontent.com/92366570/200966934-3ee0cb59-696c-4085-a8e3-f2727a5d5a78.png)

> Edit visible fields popup menu

![image](https://user-images.githubusercontent.com/92366570/200967817-e6febee6-2582-4cd0-8e94-9ae48d1ca115.png)

**Comment by lightningRalf** [comment]
I would add, that the coloring should be shown.
![image](https://github.com/community/community/assets/126403501/770bc604-36b4-4041-8365-0a74439badad)

![image](https://github.com/community/community/assets/126403501/c6ea1664-574c-4c5d-8ad9-26b125095add)


This feature could greatly reduce the confusion and need for 100 + labels, and group everything nicely with the custom fields.

**Reply by Giangpro89** [reply]
Hello
